src/pages/waiting-room-to-car/components/vehicle-details/__tests__/vehicle-details.spec.ts

```diff
@@ -0,0 +1,56 @@
+
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { VehicleDetailsComponent } from '../vehicle-details';
+import { IonicModule } from 'ionic-angular';
+import { AppModule } from '../../../../../app/app.module';
+import { By } from '@angular/platform-browser';
+import { FormGroup } from '@angular/forms';
+
+describe('VehicleDetailsComponent', () => {
+  let fixture: ComponentFixture<VehicleDetailsComponent>;
+  let component: VehicleDetailsComponent;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        VehicleDetailsComponent,
+      ],
+      imports: [
+        IonicModule,
+        AppModule,
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(VehicleDetailsComponent);
+        component = fixture.componentInstance;
+        component.formGroup = new FormGroup({});
+      });
+
+  }));
+
+  describe('DOM', () => {
+    describe('setting optional vehicle details', () => {
+      it('should dispatch a toggle school car action when school car is selected', () => {
```

Comment:
Name on this test is not inaccurate, it doesn't dispatch actions anymore.

Reply:
have changed description